docs/countme.md

```diff
@@ -25,17 +25,25 @@ standalone way.
 
 ## Disabling DNF Count Me on a system
 
-To disable this feature, you need to update the Count Me option in all RPM
-repository configuration files:
+To disable this feature, you need to stop the `rpm-ostree-countme.timer` and
+mask the corresponding unit as a precaution:
 
 ```
-sed -i 's/countme=1/countme=0/g' /etc/yum.repos.d/*.repo
+$ systemctl mask --now rpm-ostree-countme.timer
 ```
 
-You should also stop and disable the `rpm-ostree-countme.timer`:
+If you have packages layered on top of the base image from an RPM repository,
+then you will have to make sure that the `countme` option is disabled there
+until we fix [this issue in libdnf](TODO):
 
 ```
-$ systemctl disable --now rpm-ostree-countme.timer
+sed -i 's/countme=1/countme=0/g' /etc/yum.repos.d/*.repo
```

Comment:
Minor: the previous shell block uses `$` so to be consistent either we should add it here too or remove it from the previous one.